src/components/EntityInfo.vue

```diff
@@ -125,9 +132,18 @@
             <!-- Email -->
             <template v-if="businessId">
               <dt class="mr-2">Email:</dt>
-              <dd id="entity-business-email" @click="editBusinessProfile()">
+              <dd
+                id="entity-business-email"
+                :class="isHistorical ? 'pointer-events-none' : 'cursor-pointer'"
```

Comment:
Note that disabling pointer events means the click handler won't get called.

Reply:
Yes, that's the point :)  I don't want the click event to fire when the entity is historical. (I also added a safety check in the handler itself.)

Reply:
on another note, i didn't even know about the `dl`, `dt` and `dd` tags.